developer/js/source/kmlmc.ts

```diff
@@ -12,7 +12,7 @@ import { SysExits } from './util/sysexits';
 
 let inputFilename: string;
 
-/// <reference types="@keymanapp/keyman-version" />
+const KEYMAN_VERSION = require("@keymanapp/keyman-version").KEYMAN_VERSION;
```

Comment:
Ah, so it wasn't importing module-style.  I'd really thought it had been.

I'm surprised the `input-processor` tests weren't failing before this, then, since they rely on a compiled `kmlmc`.  Or maybe the failures were being hidden?

Reply:
> I'm surprised the `input-processor` tests weren't failing before this, then, since they rely on a compiled `kmlmc`. Or maybe the failures were being hidden?

I think that kmlmc was building but was probably getting `undefined` version data -- which is pretty much only reported to the user. So probably wouldn't have failed tests. Developer Server, on the other hand, wouldn't build.